README.md

```diff
@@ -1,3 +1,39 @@
 # zhyt-ios
 
+## Overview
+
+We are creating an electronic payment system that will work with our currency ("zhyt-coin") and will allow you to easily transfer money from one user to another. All you need to perform these operations is an iPhone.
+
+## Use case diagram
+
+![zhyt-coin](https://user-images.githubusercontent.com/54912616/163831250-e49168c8-bfee-4113-bc1f-ff559b081aa8.jpg)
+
+## userflow
+![userflow_zhyt](https://user-images.githubusercontent.com/54912616/164428262-8f1af5b4-ca03-4a6b-8821-223d9a916511.png)
```

Comment:
- how balance top up will work?
- editing profile does not require additional verification?
- what is user id in system? email?
- there is 2 fill registration boxes? mistaken?

Reply:
1. The balance top up is a simulation of account replenishment from own card, so you just need to enter the amount and your account balance will increase.
2. I've corrected this, I agree that it's necessary to ask additional verification.
3. Yes, email is unique
4. Yes, forgot to delete, corrected.